.travis.yml

```diff
@@ -19,9 +19,9 @@ install:
   - cd $HOME/Arduino/libraries/
   - git clone https://github.com/greiman/SdFat
   - git clone https://github.com/me-no-dev/ESPAsyncWebServer
-  - git clone -o 991f855 https://github.com/me-no-dev/ESPAsyncTCP
-  - git clone -o 6734c16 https://github.com/alanswx/ESPAsyncWiFiManager
-  - ( git clone -b 5.x https://github.com/bblanchon/ArduinoJson ; cd ArduinoJson ; git reset --hard ed98ea4 )
+  - git clone https://github.com/me-no-dev/ESPAsyncTCP
+  - git clone https://github.com/alanswx/ESPAsyncWiFiManager
+  - git clone https://github.com/bblanchon/ArduinoJson
```

Comment:
Please refer to a specific commit so that things don't suddenly break. Additionally, for this library, refer to a specific branch. This library is known to shift around the default branches from time to time, at which time everything breaks and you need to figure out why what had compiled yesterday doesn't compile today anymore. Happened twice to me.